modules/dashboard/test/TestPlan.md

```diff
@@ -17,14 +17,15 @@
 8. Check that recruitment per site view is correct (study progression panel).
    [Automate Test]
 9. Verify that for a user with 'Edit imaging browser QC status' permission the My Tasks panel reports the number of new
-   scans (i.e. the number of scans on the MRI browser page for which QC status has not been set). Check that site
+   scans (i.e. the number of scans on the MRI browser page for which QC status has not been set. Note DCC scans are not
+   counted in the query as for most projects this is not an acquisition site). Check that site
    displayed is always 'All'. Click on this task and verify that you go to the MRI Browser page.
```

Comment:
I don't think those permissions are required for this. There are permissions to restrict the list of scan to the phantom data only. 
Also, the list of tasks are based on the Edit imaging browser QC status since this is what determines how many scans need to be QC'ed.

Reply:
I see, that makes sense.